### ISSUE TITLE
HTTP ERROR 500

### ISSUE THREAD

**Issue by CptBananaPeel** [pronebird/wp-clanwars#3]
Hallo

When trying to use your plugin, which looks to be very nice - i'm getting blocked by internal error 500 constantly. 
cases of issues: 
- edit of maps from games
- adding match
I'm currently using WP version: 4.7.3

**Comment by pronebird** [comment]
Hi,

That's strange for sure. Do you use version from https://wordpress.org/plugins/wp-clanwars/ ?
The version in this repository is still under development, I apologize for messing up master copy...

**Comment by CptBananaPeel** [comment]
yes, i'm using the one you have linked.
I've tried increasing the WP Memory Limit. But no effect

**Comment by pronebird** [comment]
Do you maybe have an access to `error.log` on your server to see why you receive error 500?

**Comment by CptBananaPeel** [comment]
From the developer tool it gives me the following error messages (when trying to edit maps - in this case)
```
GET http://www.MyWebSide.com/wp-admin/admin.php?page=wp-clanwars-games&act=maps&game_id=9 500 (Internal Server Error)

/wp-admin/admin.php?page=wp-clanwars-games&act=maps&game_id=9:1
```

**Comment by CptBananaPeel** [comment]
Just noticed that the server have to run InnoDB, while mine is running MariaDB. Dont know if this might be the cause

**Comment by pronebird** [comment]
It would be great if we could get the underlying error from your developer console. Meanwhile I'll install MariaDB and poke around to see if that works which it should just like MySQL.

I don't think that InnoDB support is required anymore. When new version is ready I'll make sure to keep README up to date to reflect new requirements.

**Comment by CptBananaPeel** [comment]
when turning on WP_Debug, it gives me the following error, when trying to edit a map (same case as mentioned before)

```
Fatal error: Uncaught Error: [] operator not supported for strings in /customers/d/e/1/solidarityesport.com/httpd.www/wp-content/plugins/wp-clanwars/classes/games.class.php:79 Stack trace: #0 /customers/d/e/1/solidarityesport.com/httpd.www/wp-content/plugins/wp-clanwars/wp-clanwars.php(1276): WP_Clanwars\Games::get_game(Array) #1 /customers/d/e/1/solidarityesport.com/httpd.www/wp-includes/class-wp-hook.php(298): WP_ClanWars->on_load_manage_games('') #2 /customers/d/e/1/solidarityesport.com/httpd.www/wp-includes/class-wp-hook.php(323): WP_Hook->apply_filters('', Array) #3 /customers/d/e/1/solidarityesport.com/httpd.www/wp-includes/plugin.php(453): WP_Hook->do_action(Array) #4 /customers/d/e/1/solidarityesport.com/httpd.www/wp-admin/admin.php(212): do_action('load-clanwars_p...') #5 {main} thrown in /customers/d/e/1/solidarityesport.com/httpd.www/wp-content/plugins/wp-clanwars/classes/games.class.php on line 79
```

**Comment by pronebird** [comment]
Thanks for reporting this. Yes that's a bug. I'll notify you when I push an update.

**Comment by CptBananaPeel** [comment]
Greatly appreciated!
And thanks a bunch for the quick response, impressive!

**Comment by pronebird** [comment]
The patch has been released with v1.7.2. The update should be available shortly via Plugins menu in your wp-admin. I've tested the patch and it works. However, re-open this issue if it does not work for you for some reason.

Thanks again for reporting this. The same code worked differently on php 7 and caused fatal error to happen. I wouldn't spot that early enough myself since I only used PHP 5 until today. :)
